onadata/apps/api/tests/viewsets/test_xform_viewset.py

```diff
@@ -3539,6 +3540,32 @@ def test_export_form_data_async(self, async_result):
                 export = Export.objects.get(task_id=task_id)
                 self.assertTrue(export.is_successful)
 
+    @override_settings(CELERY_TASK_ALWAYS_EAGER=True)
+    def test_raises_exception_for_xls_export_formats(self):
```

Comment:
Test name is a bit misleading; I don't see an exception raised in this test

Reply:
The tests asserts the `ExportTypeError` is raised with .xls formats [here](https://github.com/onaio/onadata/pull/2346/files#diff-a7dc39d301394b01b2cfcd02e9dcfc7b3eadca03097090cd2b3a5bf72fa1cdf6R3554)